From a539f8ceb546df1892274d7dfafddee896e7e96d Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinh.nguyen@intel.com>
Date: Fri, 2 Mar 2018 09:50:14 -0600
Subject: [PATCH 20/25] FogBugz #539974: set clocks for watchdog timer on
 Stratix10

commit  b002bf552cb779951ca843a058545dc146989270 from
https://github.com/altera-opensource/linux-socfpga.git

Populate the clocks property in the watchdog timers on Stratix10.

Signed-off-by: Dinh Nguyen <dinh.nguyen@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
index c8bc964..dc0a4a6 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
@@ -882,6 +882,7 @@
 			reg = <0xffd00200 0x100>;
 			interrupts = <0 117 4>;
 			resets = <&rst WATCHDOG0_RESET>;
+			clocks = <&l4_sys_free_clk>;
 			status = "disabled";
 		};
 
@@ -890,6 +891,7 @@
 			reg = <0xffd00300 0x100>;
 			interrupts = <0 118 4>;
 			resets = <&rst WATCHDOG1_RESET>;
+			clocks = <&l4_sys_free_clk>;
 			status = "disabled";
 		};
 
@@ -898,6 +900,7 @@
 			reg = <0xffd00400 0x100>;
 			interrupts = <0 125 4>;
 			resets = <&rst WATCHDOG2_RESET>;
+			clocks = <&l4_sys_free_clk>;
 			status = "disabled";
 		};
 
@@ -906,6 +909,7 @@
 			reg = <0xffd00500 0x100>;
 			interrupts = <0 126 4>;
 			resets = <&rst WATCHDOG3_RESET>;
+			clocks = <&l4_sys_free_clk>;
 			status = "disabled";
 		};
 
-- 
2.8.1

